src/sonic-yang-models/yang-models/sonic-types.yang

```diff
@@ -107,4 +111,53 @@ module sonic-types {
             pattern "percentage|used|free|PERCENTAGE|USED|FREE";
         }
     }
+
+    typedef oper-status {
+        type enumeration {
+            enum unknown;
+            enum up;
+            enum down;
+        }
+		description "Operational status of an entity such as Port, MCLAG etc";
+    }
+
+    typedef mode-enable {
+        type enumeration {
+            enum enable;
```

Comment:
mode-enable with values enable/disable..doesn't look good, suggest renaming as 'status' or 'mode' with enable/disable?

Reply:
Done. Changed to mode-status.